docs/releases/1.19-NOTES.md

```diff
@@ -74,6 +74,20 @@ has been updated by a newer version of kops unless it is given the `--allow-kops
 
 * See note about [Openstack Cinder plugin](#openstack-cinder-plugin) above.
 
+* Terraform users, in order to prevent downtime you will have to remove the state of the existing ELB attatchments from your terraform state file. This is due to us migrating to setting the loadbalancer attatchment when defining the ASG resource. The migration was required due to a bug described in #9913.
```

Comment:
```suggestion
* Terraform users, in order to prevent downtime you will have to remove the state of any existing ELB or TargetGroup attatchments from your Terraform state file. This is due to migrating the attachments to the in-line `aws_autoscaling_group` fields. See the [terraform documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/autoscaling_group) for more information about the difference. This migration is required due to a bug described in [#9913](https://github.com/kubernetes/kops/issues/9913).
```